selenium_tests/lti_emailer/page_objects/emailer_permission_po.py

```diff
@@ -0,0 +1,44 @@
+__author__ = 'hic048'
+
+from selenium.webdriver.common.by import By
+from selenium_tests.lti_emailer.page_objects.emailer_base import EmailerBasePage
+
+
+# These are page-specific locators.
+class Locator2(object):
+      _emailer_url = "https://canvas.icommons.harvard.edu/courses/6389/external_tools/1759"
+      _confirm_mask = (By.CSS_SELECTOR, "a.btn.masquerade_button")
+      _unauthstate = (By.CSS_SELECTOR, "h2.ui-state-error")
+      _authstate = (By.CSS_SELECTOR, "h2")
+      _switchframe = (By.ID, "tool_content")
+      _pageload = (By.CSS_SELECTOR, "h2")
+
+
+
+class PermissionPage(EmailerBasePage):
+
+    def switch_frame(self):
+        driver = self._driver
+        driver.switch_to.frame(driver.find_element(*Locator2._switchframe))
```

Comment:
The `BasePageObject` has a method `.focus_on_tool_frame()` which should be doing the same thing -- I think we can just use that wherever this `switch_frame()` is called, and remove this method from PermissionPage altogether.